tests/tests/client-unmount-recovery.sh

```diff
@@ -53,6 +53,9 @@ while [ "$SECONDS" -lt "$END" ]; do
 
         for i in "${lock_arr[@]}"; do
                 if [[ ! " ${rid_arr[*]} " =~ " $i " ]]; then
+                        echo -e "RID($i) exists" >> $T_TMP.log
+                        echo -e "rid_arr:\n${RID_LIST[@]}" >> $T_TMP.log
```

Comment:
The _LIST vars are just strings with whitespace separating the entries.  It's the read -a that turns them into arrays.  So either $RID_LIST or ${rid_arr[@]}.

Reply:
ok, updated.